ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/as-path.j2

```diff
@@ -0,0 +1,24 @@
+{% if as_path.access_lists is arista.avd.defined %}
+
+## AS Path Lists
+
+### AS Path Lists Summary
+{%     if as_path.regex_mode is defined %}
+
+AS Path Regex Mode is **{{ as_path.regex_mode }}**.
+{%     endif %}
+
+| List Name | Type | Match | Origin |
+| --------- | ---- | ----- | ------ |
+{%     for as_path_access_list in as_path.access_lists | arista.avd.natural_sort %}
+{%         for as_path_access_list_entry in as_path_access_list.entries | arista.avd.natural_sort %}
+| {{ as_path_access_list.name }} | {{ as_path_access_list_entry.type }} | {{ as_path_access_list_entry.match | replace('|', '\|') }} | {{ as_path_access_list_entry.origin | arista.avd.default('any') }} |
```

Comment:
Line 13 -> natural_sort will only take care if ```as_path.access_lists``` is not none.
Line 14 -> natural_sort will only take care if ```as_path_access_list.entries``` is not none. If entries has only (-type: <> then it's not none, then we need handle other knobs with arista.avd.default('-'))

So we still need to take care of ```as_path_access_list.name```, ```as_path_access_list_entry.type, match and origin```

Reply:
We should _not_ use `natural_sort` on `entries` on ACLs. The order is important, and we should let the user decide on which order the entries will be configured. So please revert.

Reply:
got it. let me revert this but add a condition to make sure that at least type and match are defined.